lib/lightning_web/templates/layout/live.html.heex

```diff
@@ -2,17 +2,35 @@
   <div class="flex flex-row h-full">
     <nav class="basis-48 bg-primary text-white" style="width: 300px">
       <div class="flex flex-col h-full">
-        <div class="w-full h-20 bg-primary flex items-center justify-center">
-          <%= live_redirect(to: Routes.dashboard_index_path(@socket, :index)) do %>
-            <img
-              class="h-10 w-10"
-              src={
-                Routes.static_path(@socket || @conn, "/images/logo-white.svg")
-              }
-              alt="OpenFn"
-            />
-          <% end %>
-        </div>
+        <%= if assigns[:project] do %>
+          <div class="mt-4 pb-2 mb-4 text-black-300 bg-primary">
+            <%= live_redirect(class: "px-4 py-3", to: Routes.project_dashboard_index_path(@socket, :show, @project.id)) do %>
+              <img
+                class="h-6 w-6 inline-block align-middle"
+                src={
+                  Routes.static_path(@socket || @conn, "/images/logo-white.svg")
+                }
+                alt="OpenFn"
+              />
+            <% end %>
+            <select
```

Comment:
@taylordowns2000 I will need your help on the stylization of the project picker. Here is the part where the select field is. The major problem I faced on it is when the project name is long, the select field goes below the OpenFn logo (which is not corresponding to the design [here](https://www.figma.com/file/ddg6ahCGnq2nPYIurpnoHF/OpenFN-High-Fidelity?node-id=1%3A3464)). Thanks in advance for helping on that.